playwright/tests/pageobjects/abstract/Table.ts

```diff
@@ -15,13 +17,34 @@ export class Table {
     return new Row(this.rows, index);
   }
 
+  async focusTable() {
+    await this.table.focus();
+  }

```

Comment:
I would not create such small, simple methods. Rename the locator `table` to `locator` and just call the `focus` method directly on it if needed. E.g.:

```typescript
await editor.table.locator.focus();